README.md

```diff
@@ -116,23 +123,23 @@ What is an IIFE, and why might you use it?
 
 **Your answer:**
 
-> ...
+> An IIFE is an "immediately invoked function expression."  It is a function that is called as soon as it's defined, and any variables declared inside it will only exist inside it and will not pollute other scopes.   
 
 ## Question 7
 
 What is the `ui-sref` directive, and how is it used?
 
 **Your answer:**
 
-> ...
+> The "ui-sref" directive is part of the Angular ui-router and is used to generate HTML links from Angular structures and data.  
```

Comment:
:+1: and they cause a view to be rendered that is associated with a particular state.